seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-jdbc/src/main/java/org/apache/seatunnel/flink/source/JdbcSource.java

```diff
@@ -71,19 +80,17 @@
     private String password;
     private int fetchSize = DEFAULT_FETCH_SIZE;
     private Set<String> fields;
+    private int parallelism = 1;
```

Comment:
I find in the former code, the default value of parallelism in flink is -1. Why we change to 1?